core/src/test/java/com/linecorp/armeria/client/DnsMetricsTest.java

```diff
@@ -195,24 +196,15 @@ void dns_test_no_answer() throws ExecutionException, InterruptedException {
 
     @Test
     void test_with_real_dns_query() throws ExecutionException, InterruptedException {
-        final MeterRegistry pm1 = PrometheusMeterRegistries.newRegistry();
 
-        try (ClientFactory factory = ClientFactory.builder()
-                .meterRegistry(pm1)
-                .build()) {
+        try (ClientFactory factory = ClientFactory.builder().build()) {
```

Comment:
You can still use `PrometheusMeterRegistries.newRegistry()` here because this class is a test class.